python/paddle/fluid/tests/book/high-level-api/word2vec/no_test_word2vec_new_api.py

```diff
@@ -80,8 +80,11 @@ def inference_program(is_sparse):
 
 
 def train_program(is_sparse):
-    next_word = fluid.layers.data(name='nextw', shape=[1], dtype='int64')
+    # The declaration of 'next_word' must be after the invoking of inference_program,
+    # or the data input order of train program would be [next_word, firstw, secondw,
+    # thirdw, forthw], which is not correct.
     predict_word = inference_program(is_sparse)
```

Comment:
So is it accurate to say that "the very line of train_program should be getting the infererence_program"?